apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/WriteTests.cs

```diff
@@ -0,0 +1,170 @@
+        public async Task WriteNulls()
+        {
+            bool readData;
+            Assert.True(await Insert(new SpannerParameterCollection
+                        {
+                            new SpannerParameter("BoolValue", SpannerDbType.Bool, null),
+                            new SpannerParameter("Int64Value", SpannerDbType.Int64, null),
+                            new SpannerParameter("Float64Value", SpannerDbType.Float64, null),
+                            new SpannerParameter("StringValue", SpannerDbType.String, null),
+                            new SpannerParameter("TimestampValue", SpannerDbType.Timestamp, null),
+                            new SpannerParameter("DateValue", SpannerDbType.Date, null),
+                            new SpannerParameter("BoolArrayValue", SpannerDbType.ArrayOf(SpannerDbType.Bool), null),
+                            new SpannerParameter("Int64ArrayValue", SpannerDbType.ArrayOf(SpannerDbType.Int64), null),
+                            new SpannerParameter("Float64ArrayValue", SpannerDbType.ArrayOf(SpannerDbType.Float64), null),
+                            new SpannerParameter("StringArrayValue", SpannerDbType.ArrayOf(SpannerDbType.String), null),
+                            new SpannerParameter("BytesArrayValue", SpannerDbType.ArrayOf(SpannerDbType.Bytes), null),
+                            new SpannerParameter("TimestampArrayValue", SpannerDbType.ArrayOf(SpannerDbType.Timestamp), null),
+                            new SpannerParameter("DateArrayValue", SpannerDbType.ArrayOf(SpannerDbType.Date), null),
+                        })
+                        == 1);
+            using (var reader = await GetLastRowAsync())
+            {
+                readData = true;
+                Assert.True(null == reader.GetValue(reader.GetOrdinal("BoolValue")));
+                Assert.True(null == reader.GetValue(reader.GetOrdinal("Int64Value")));
+                Assert.True(null == reader.GetValue(reader.GetOrdinal("Float64Value")));
+                Assert.True(null == reader.GetValue(reader.GetOrdinal("StringValue")));
+                Assert.True(null == reader.GetValue(reader.GetOrdinal("TimestampValue")));
+                Assert.True(null == reader.GetValue(reader.GetOrdinal("DateValue")));
+                Assert.True(null == reader.GetValue(reader.GetOrdinal("BoolArrayValue")));
+                Assert.True(null == reader.GetValue(reader.GetOrdinal("Int64ArrayValue")));
+                Assert.True(null == reader.GetValue(reader.GetOrdinal("Float64ArrayValue")));
+                Assert.True(null == reader.GetValue(reader.GetOrdinal("StringArrayValue")));
+                Assert.True(null == reader.GetValue(reader.GetOrdinal("BytesArrayValue")));
+                Assert.True(null == reader.GetValue(reader.GetOrdinal("TimestampArrayValue")));
+                Assert.True(null == reader.GetValue(reader.GetOrdinal("DateArrayValue")));
+            }
+            Assert.True(readData);
```

Comment:
This is unnecessary. If you got to this point, you necessary also went through the using statement and therefore set `readData` to true.